style.json

```diff
@@ -230,15 +230,172 @@
         ]
       }
     },
+    {
+      "id": "waterway-bridge-case",
```

Comment:
Now waterway bridges are drawn under buildings, roads, etc.:
![image](https://user-images.githubusercontent.com/2271850/35214812-b297921e-ff62-11e7-8a0a-4d4c32fb0545.png)

Please move two bridge layers (`waterway-bridge-case` and `waterway-bridge`) prior to other bridge layers, more precisely between `railway` and `bridge_minor case` layers.